_data/browsers.yml

```diff
@@ -1,24 +1,20 @@
 - firefox:
   vendor: Mozilla
   name: Firefox
-  version: 52
 
 - chrome:
   vendor: Google
   name: Chrome
-  version: 63
 
 - edge:
   vendor: Microsoft
   name: Edge
-  version: 16
 
-- ios:
+- safari:
   vendor: Apple
   os: iOS
   name: Safari
   version: 10.3
```

Comment:
Did you keep that version intentionally, or just forgot to remove it?

Reply:
No, it's intentional. It's a manual minimum version. If it's higher than the computed one, it sticks. It's this version because of the certificate issues that exist on macOS Safari < 10.3.

(I also moved from referring to iOS for Safari to just Safari.)

Reply:
Here's the bit of comparison code to support manual versions:

https://github.com/cockpit-project/cockpit-project.github.io/pull/270/files#diff-4283f68ad6ffc7bb5e893943e7f1b7dfR55-R67

It's a little bit of a mess, but that's because I'm doing this in liquid (which is the right place for this, in this case). I basically just compare the two values as floats.

Reply:
Makes sense, thanks! Can you please add a comment about it, so that we can tell our future selves if/when to bump it?

Reply:
It's a separate commit of the 3 and now has a bunch of text in the commit to explain it better.